src/app.js

```diff
@@ -1 +1,22 @@
 'use strict';
+
+const fs = require('fs');
+
+function funcRead() {
+  const terminal = process.argv.slice(2);
+  const filePath = './src/' + terminal[0];
+  const newFilePath = './src/' + terminal[1];
+  const isFile = fs.existsSync(filePath);
+
+  if (isFile) {
+    const fileInfo = fs.readFileSync(filePath, 'utf8');
+
+    fs.writeFile(newFilePath, fileInfo, (err, data) => {
```

Comment:
The 'data' argument in the callback seems unnecessary, since 'fs.writeFile' doesn't provide a second argument to its callback. You should remove it to avoid confusion.